Ship_Game/GameScreens/ScreenManager.cs

```diff
@@ -653,6 +654,22 @@ void PerformHotLoadTasks(UpdateTimes elapsed)
             }
         }
 
+        public void StartMusic(string musicName)

```

Comment:
This wont restart the music when the sound track is done. there is a bug here (Ambient music or shipyard)